TokenInfo.py

```diff
@@ -1,46 +1,8 @@
-"""File containing token names and decimals."""
+import requests
+import json
 
-TOKEN_NAMES = {
-    'T0000': 'OWL',
-    'T0001': 'WETH',
-    'T0002': 'USDT',
-    'T0003': 'TUSD',
-    'T0004': 'USDC',
-    'T0005': 'PAX',
-    'T0006': 'GUSD',
-    'T0007': 'DAI',
-    'T0008': 'sETH',
-    'T0009': 'sUSD',
-    'T0010': 'sBTC',
-    'T0011': 'WBTC',
-    'T0012': 'SAI',
-    'T0013': 'cDAI',
-    'T0014': 'aDAI',
-    'T0015': 'SNX',
-    'T0016': 'CHAI',
-    'T0017': 'PAXG',
-    'T0018': 'GNO'
-}
 
-
-TOKEN_DECIMALS = {
-    'T0000': 18,
-    'T0001': 18,
-    'T0002': 6,
-    'T0003': 18,
-    'T0004': 6,
-    'T0005': 18,
-    'T0006': 2,
-    'T0007': 18,
-    'T0008': 18,
-    'T0009': 18,
-    'T0010': 18,
-    'T0011': 8,
-    'T0012': 18,
-    'T0013': 8,
-    'T0014': 18,
-    'T0015': 18,
-    'T0016': 18,
-    'T0017': 18,
-    'T0018': 18
-}
+def get_token_data():
+    token_list_url = "https://raw.githubusercontent.com/gnosis/dex-js/master/src/tokenList.json"
```

Comment:
Two questions to this:
* Currently, not all tokens that are listed in the contract are part of that list. Is this actively being maintained?
* Does this assume that tokens are listed in the same order on the mainnet and rinkeby deployments?

Reply:
Yes, the other tokens will very likely be added, otherwise, I will remove this part.
And currently, it does assume that we have tokens in the same order. I already made an issue for that: https://github.com/gnosis/dex-js/issues/113
This is also not acceptable for front-end and the other parts, where it is used.